### DISCUSSION TITLE
cant load instruments

### DISCUSSION THREAD

**Discussion by Mblimassol** [apohl79/audiogridder#398] [Help]
First post here!
Congratulations to developers for this amazing little software. its a must. 

Everything ok here with windows 10 pro on both machines with i9 9900k, 64g ram and a lot of SSD's and Cubase 11 pro on main pc. 
a little issue with instruments. I cant load anything, vst2 or 3 on both AG versions vst2 or 3. 
it gives me the attached message. 
any thought?
![Capture](https://user-images.githubusercontent.com/79401816/108623009-2eb34c80-7445-11eb-8732-74f3cf5640e3.PNG)

**Comment by apohl79** [comment]
> First post here!

Welcome :-)

> Congratulations to developers for this amazing little software. its a must. 

thx!


> Everything ok here with windows 10 pro on both machines with i9 9900k, 64g ram and a lot of SSD's and Cubase 11 pro on main pc. 
> 
> a little issue with instruments. I cant load anything, vst2 or 3 on both AG versions vst2 or 3. 
> 
> it gives me the attached message. 
> 
> any thought?

This message means the server did not respond. This means most of the time that it crashed when trying to load a plugin. Please check the server log.

**Comment by Mblimassol** [comment]
I dont know what to check on log file. Maybe you can do for me?
[AudioGridderServer_2021-02-21_13-26-52.log](https://github.com/apohl79/audiogridder/files/6017721/AudioGridderServer_2021-02-21_13-26-52.log)

**Reply by Mblimassol** [reply]
though it hapens only with instruments. 
Fx are ok

**Reply by apohl79** [reply]
ok strange, I don't see a single attempt to load the plugin in the server log.

Can you follow the bug reporting steps please (https://audiogridder.com/bug-reports/) and create possibly a single server logfile and a single plugin logfile for the session (as well as trace files) and share them?

Thx

**Reply by apohl79** [reply]
As in my last comment, please follow the bug reporting steps, so i can see server an client log and trace files. thx.

**Comment by Mblimassol** [comment]
check the trace and log files here please
https://www.dropbox.com/s/02qn00ipl89s11n/aglog%20files.rar?dl=0

**Comment by Mblimassol** [comment]
[Desktop.zip](https://github.com/apohl79/audiogridder/files/6215798/Desktop.zip)
